project/accounts/templates/accounts/register/entry_base.html

```diff
@@ -1,24 +1,23 @@
+{% extends "base.html" %}
+{% load static %}
 {% load i18n %}
-<html>
-<head>
-    {% load static %}
-    {% include "accounts/base/links.html" %}
+
+{% block extra_css %}
     {% include "accounts/base/less_headers/login_less.html" %}
-    <title>{% block title %}CiviWiki{% endblock %}</title>
-</head>
-<body>
+{% endblock extra_css %}
+
+{% block content %}
 {% include 'accounts/register/header.html' %}
 <div class="entry-base center-align valign-wrapper">
     <div class="entry-forms">
         <div class="card wrapper z-depth-3">
                 <div class="card-content" id="entry-content">
-                    {% block content %}{% endblock %}
+                    {% block entry_content %}{% endblock %}
```

Comment:
This should remain as `block content`
```suggestion
                    {% block content %}{% endblock %}
```